index.html

```diff
@@ -738,8 +738,8 @@ <h5>Fade both video and audio streams</h5>
       <dl>
           <dt>ffmpeg</dt><dd>starts the command</dd>
           <dt>-i <em>input_file</em></dt><dd>path, name and extension of the input file</dd>
-          <dt>-filter:v "fade=in:st=0:d=1, fade=out:st=59:d=1"</dt><dd>applies a video filter that fades in the first and last 30 video frames</dd>
-          <dt>-filter:a "afade=in:st=0:d=1, afade=out:st=59:d=1"</dt><dd>applies an audio filter that fades in the first and last 30 video frames. <code>st</code> sets the start and <code>d</code> sets the duration.</dd>
+          <dt>-filter:v "fade=in:st=0:d=1, fade=out:st=59:d=1"</dt><dd>applies a video filter that fades in the first and last 30 video frames. <code>st</code> sets the start and <code>d</code> sets the duration. <code>st=59:d=1</code> is equivalent to one second</dd>
+          <dt>-filter:a "afade=in:st=0:d=1, afade=out:st=59:d=1"</dt><dd>applies an audio filter that fades in the first and last 30 video frames and 44100 audio samples. <code>st</code> sets the start and <code>d</code> sets the duration.<code>st=59:d=1</code> is equivalent to one second</dd>
```

Comment:
Again, I'd remove references to frames here as the code is talking about duration. I'd also remove the reference to 44100 audio samples, as this would only be accurate if the source video is 44kHz, when a lot of video ends up being 48kHz or something else. I'd repeat the clarification about the value `59` referring to this being a 60 second source video.